riscv/mmu.h

```diff
@@ -187,6 +211,21 @@ class mmu_t
     })
   }
 
+  // for shadow stack amoswap
+  template<typename T>
+  T ssamoswap(reg_t addr, reg_t value) {
+    convert_load_traps_to_store_traps({
```

Comment:
The `convert_load_traps_to_store_traps` wrapper is not needed since load with `ss_access` already only causes store-*-fault.